packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2474,6 +2483,8 @@ export default class Meeting extends StatelessWebexPlugin {
         switch (newShareStatus) {
           case SHARE_STATUS.REMOTE_SHARE_ACTIVE: {
             const sendStartedSharingRemote = () => {
+              this.shareInstanceId = contentShare.shareInstanceId;
```

Comment:
I'm not convinced if it's a good idea to be setting `Meeting.shareInstanceId` here as we're setting it to the remote share instance. I'm worried that there may be some race conditions if someone starts sharing at the same time as we are and this may overwrite the value that we set in `publishStreams()`. After all I think we only use `Meeting.shareInstanceId` for sending events about our share so I don't think we need to store the remote share instance id at all in `Meeting.shareInstanceId`

Reply:
What do you suggest we do for `client.media.rx.start` and `client.media.rx.stop` then? We would want the contentshare.instanceId in that case right? Not a super big problem for now, since this metric isn't turned on for share.

Reply:
yes, you're right we need it in that case, but it would be better if we store it as a separate `remoteShareInstanceId` for that, or maybe we don't even need to to store it separately, because I think this stuff is already stored - see MediaSharesUtils.getShareInstanceId()